app/src/test/kotlin/accessibility/reporting/tool/UpdateReportTest.kt

```diff
@@ -118,36 +124,394 @@ class UpdateReportTest {
         dummyreport.assertExists(updatedJsonResponse)
         updatedJsonResponse["successCriteria"].toList().assert {
             size shouldBe 1
-            val criterion=this.first()
-            criterion ["name"].asText() shouldBe "updated criterion"
-            criterion ["description"].asText() shouldBe "updated description"
-            criterion ["principle"].asText() shouldBe "updated principle"
-            criterion ["guideline"].asText() shouldBe "updated guideline"
-            criterion ["tools"].asText() shouldBe "updated tools"
-            criterion ["number"].asText() shouldBe "updated number"
-            criterion ["breakingTheLaw"].asText() shouldBe "updated breakingTheLaw"
-            criterion ["lawDoesNotApply"].asText() shouldBe "updated lawDoesNotApply"
-            criterion ["tooHardToComply"].asText() shouldBe "updated tooHardToComply"
-            criterion ["contentGroup"].asText() shouldBe "updated contentGroup"
-            criterion ["status"].asText() shouldBe "COMPLIANT"
-            criterion ["wcagLevel"].asText() shouldBe "AA"
+            val criterion = this.first()
+            criterion["name"].asText() shouldBe "updated criterion"
+            criterion["description"].asText() shouldBe "updated description"
+            criterion["principle"].asText() shouldBe "updated principle"
+            criterion["guideline"].asText() shouldBe "updated guideline"
+            criterion["tools"].asText() shouldBe "updated tools"
+            criterion["number"].asText() shouldBe "updated number"
+            criterion["breakingTheLaw"].asText() shouldBe "updated breakingTheLaw"
+            criterion["lawDoesNotApply"].asText() shouldBe "updated lawDoesNotApply"
+            criterion["tooHardToComply"].asText() shouldBe "updated tooHardToComply"
+            criterion["contentGroup"].asText() shouldBe "updated contentGroup"
+            criterion["status"].asText() shouldBe "COMPLIANT"
+            criterion["wcagLevel"].asText() shouldBe "AA"
+
+        }
+    }
+
+    //Lag en test hvor bare ett metadatafelt blir send med (f.eks team eller descriptive name)
+    @Test
+    fun `partial updates metadata`() = setupTestApi(database) {
+        val newDescriptiveName = "Updated Report Title"
+        val updateDescriptiveName = """
+        {
+            "reportId": "${dummyreport.reportId}",
+            "descriptiveName": "${newDescriptiveName}"
+        }
+    """.trimIndent()
+
+        val response5 = client.patchWithJwtUser(testUser, "api/reports/${dummyreport.reportId}/update") {
```

Comment:
```suggestion
        val updateNameResponse = client.patchWithJwtUser(testUser, "api/reports/${dummyreport.reportId}/update") {
```

Gi responsevariablene litt mer gjennkjenbare navn (da er det også enklere å ikke bruke feil senere 😊 )

Reply:
har prøvd å ha gjennkjenbøare navn.